### PR TITLE
Create accessibility.yml

### PR DESCRIPTION
**Pull request by LauraLangdon**
Adding accessibility checks to docs workflow.

I'm not sure whether we can use this without getting an API key (it's listed as an optional input, so it's not clear). If we do we'd need to subscribe. No idea whether this is our best option.

### CODE REVIEW

File: .github/workflows/accessibility.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+- name: A11yWatch - The Accessibility Suite
+  uses: a11ywatch/github-action@v1.2.44
+  with:
+    WEBSITE_URL: ${{ secrets.https://docs.suborbital.dev }}
```
Comment (flaki) [inline .github/workflows/accessibility.yml:4]: This does not look right :thinking:

**Comment by flaki** [comment]
> I'm not sure whether we can use this without getting an API key (it's listed as an optional input, so it's not clear). If we do we'd need to subscribe. No idea whether this is our best option.

It looks like free scans show only a limited output:
image

According to the [pricing page](https://a11ywatch.com/#plans-section), for one website one could use the free tier, but it's limited to 2 manual scans / 3 API requests a day. This would only become a problem if we started using this on every pull request (preview deploy), which would be inarguable the ideal usage (checking for a11y issues/regressions in preview before the merge).

If we end up implementing this (this can be a followup), we can use the basic plan ($120/yr), a 100 requests/day should be enough for a while.

**Comment by LauraLangdon** [comment]
Update: I've created an account and have an API token; just need to get a secret set up for it to add to the workflow.

File: .github/workflows/accessibility.yml (hunk 1 above)
Comment (LauraLangdon) [reply .github/workflows/accessibility.yml:4]: > This does not look right 🤔

🤷‍♀️ I just followed this: https://github.com/marketplace/actions/a11ywatch-the-accessibility-suite

File: .github/workflows/accessibility.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
```
Comment (arbourd) [inline .github/workflows/accessibility.yml:4]:
```suggestion
    WEBSITE_URL: https://docs.suborbital.dev
```

File: .github/workflows/accessibility.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    FAIL_ERROR_COUNT: 1
+    A11YWATCH_TOKEN: A11YWATCH_TOKEN
```
Comment (arbourd) [inline .github/workflows/accessibility.yml:6]:
```suggestion
    A11YWATCH_TOKEN: ${{ secrets.A11YWATCH_TOKEN }} 
```

File: .github/workflows/accessibility.yml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+- name: A11yWatch - The Accessibility Suite
+  uses: a11ywatch/github-action@v1.2.44
+  with:
+    WEBSITE_URL: ${{ secrets.https://docs.suborbital.dev }}
+    FAIL_ERROR_COUNT: 1
+    A11YWATCH_TOKEN: ${{ secrets.A11YWATCH_TOKEN }}
```
Comment (arbourd) [inline .github/workflows/accessibility.yml:6]:
```suggestion
name: a11y

on:
  pull_request:
  push:
    branches: 
      - main

jobs:
  test:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v3
	  - uses: a11ywatch/github-action@v1.2.44
	     with:
	       WEBSITE_URL: ${{ secrets.https://docs.suborbital.dev }}
	       FAIL_ERROR_COUNT: 1
	       GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
	       A11YWATCH_TOKEN: ${{ secrets.A11YWATCH_TOKEN }} 
```

File: .github/workflows/accessibility.yml [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: a11ywatch/github-action@v1.2.44
+         with:
+           WEBSITE_URL: https://docs.suborbital.dev
+           FAIL_ERROR_COUNT: 1
+           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+           A11YWATCH_TOKEN: ${{ secrets.A11YWATCH_TOKEN }}
```
Comment (arbourd) [inline .github/workflows/accessibility.yml:19]:
```suggestion
      - uses: a11ywatch/github-action@v1.2.44
        with:
          WEBSITE_URL: https://docs.suborbital.dev
          FAIL_ERROR_COUNT: 1
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
          A11YWATCH_TOKEN: ${{ secrets.A11YWATCH_TOKEN }} 
```

File: .github/workflows/accessibility.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: a11ywatch/github-action@v1.2.44
+        with:
+          WEBSITE_URL: https://docs.suborbital.dev
+          FAIL_ERROR_COUNT: 1
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (arbourd) [inline .github/workflows/accessibility.yml:18]:
```suggestion
          TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/accessibility.yml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+  push:
+    branches: 
+      - main
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: a11ywatch/github-action@v1.2.44
```
Comment (arbourd) [inline .github/workflows/accessibility.yml:14]:
```suggestion
      - uses: a11ywatch/github-action@v1.2.43
```

**Review by arbourd — COMMENTED** [review]
hopefully the last one

File: .github/workflows/accessibility.yml (hunk 6 above)
Comment (arbourd) [reply .github/workflows/accessibility.yml:18]: this was my mistake

File: .github/workflows/accessibility.yml (hunk 7 above)
Comment (arbourd) [reply .github/workflows/accessibility.yml:14]: 44 doesn't exist yet, i opened a PR on their side to fix their docs

Comment (LauraLangdon) [reply .github/workflows/accessibility.yml]: Yep, just saw the same and pushed a new commit for this PR!

**Comment by j-mendez** [comment]
Hi I am the creator of A11yWatch. Let me know if you need help with anything! @arbourd @LauraLangdon

**Comment by ramonh** [comment]
> Hi I am the creator of A11yWatch. Let me know if you need help with anything! @arbourd @LauraLangdon

Thank you so much, @j-mendez!

**Comment by j-mendez** [comment]
@LauraLangdon Sorry for the confusion on the words for the rates and plans. Currently the single scan or api endpoint `/scan-simple` is free for usage behind rate limits. Prior there was not any `authentication` detection for the endpoint. We made the CLI to handle the auth before we added the auth to the API endpoint usage which is our faults. Just added `authentication` to `/scan-simple` so that you can get the full response instead of the limited one. The API usage limits is actually only tied to the full site-wide scans that you perform in the dashboard with the dropdown scan option or upon adding your website. The full site-wide `crawl` API will be released public soon with the next cli release. Sorry for the confusion again and thank you for helping/using the app!

**Pull request merged**